lib/extendedkey.ex

```diff
@@ -220,7 +219,7 @@ defmodule Bitcoinex.ExtendedKey do
   def get_parent_fingerprint(%__MODULE__{parent_fingerprint: pfp}), do: pfp
 
   @spec get_fingerprint(t()) :: binary
-  def get_fingerprint(xkey = %__MODULE__{}) do
+  def get_fingerprint(xkey) do
```

Comment:
would prefer to keep these. If we change the order to `%__MODULE__{} = xkey` credo should be happy